wiki/en/Running-a-Server.md

```diff
@@ -125,185 +148,176 @@ To register with one of the Directories built into the Jamulus Client, replace `
+To turn recording on or off (depending on the current state):
 
-### Recording Directory
+~~~
+ [Unit]
+ Description=Toggle recording state of Jamulus Server
+ Requisite=Jamulus-Server
 
-This sets the path to where the Server's recordings will be stored. With this path set, the "Enable Jam Recorder" function in the Server Setup tab will make recording start once the first person connects to the Server, and stops when the last person leaves. Use the "New Recording" button to create a new sub-directory in which the recordings will be stored from then on. Note that Recordings are per track in [Audacity](https://www.audacityteam.org/) `.lof` format and [REAPER](https://en.wikipedia.org/wiki/REAPER) `.rpp`. Open the respective files to listen to them in those applications.
+ [Service]
+ Type=oneshot
+ ExecStart=/bin/systemctl kill -s SIGUSR2 Jamulus-Server
+~~~
 
-**Note**: When your Server is recording, Clients will display a message that recording is on.
+To start a new recording:
 
-### Custom Directory Address
+~~~
+ [Unit]
+ Description=Start a new recording on Jamulus Server
+ Requisite=Jamulus-Server
 
-Leave this field empty unless you need to list your Server on a [Custom Directory](#3-directory) or run a Directory.
+ [Service]
+ Type=oneshot
+ ExecStart=/bin/systemctl kill -s SIGUSR1 Jamulus-Server
+~~~
 
-### Server List Filename
+_Note: The Jamulus service name in the `ExecStart` line needs to be the same as the `.service` file name you created when setting systemd to control your Jamulus Server. So in this example it would be `Jamulus-Server.service`_
 
-Leave this field empty unless you need to run your Server as a [Directory](#3-directory). When in use, this holds the list of registered Servers whilst restarting the Directory. This prevents the server list appearing "empty" until the Servers re-register.
+Run `sudo systemctl daemon-reload` to register them for first use.
 
-### Delay panning
+Now you can run these with the `systemctl` command, for example:
 
-This option uses small differences in sound arrival time between the two ears. It produces a stereo effect similar to natural human hearing when compared to normal "volume" panning.
+`sudo systemctl start jamulusTogglerec` 
 
-### Start Minimised
+You can see the result of these commands if you run `systemctl status jamulus`, or by viewing the logs.
 
-**Windows users** - If you want the Server to start automatically on system start, enable the corresponding check box.
+---
 
-### Server status icon
+## Servers on the desktop
 
-When the Server is running, the operating system will show an icon in the system tray or status area to represent whether the Server is active:  
+Jamulus can be run in Server mode from the desktop. This gives you a graphical user interface to control most of the settings. To do this, first [install Jamulus](Getting-Started) for your platform, the do one of the following:
```

Comment:
```suggestion
Jamulus can run in Server mode on your desktop computer. This gives you a graphical user interface to control most of the settings. To do this, first [install Jamulus](Getting-Started) for your platform, the do one of the following:
```

Reply:
> desktop computer

This doesn't mean what "from your desktop" is intended to mean.  A "desktop computer" is used to mean something other than a phone, tablet, laptop or server.  Here, the manual is referring to a GUI - regardless of the computer platform.

Reply:
How about _on your local computer_? Or _home computer_?

Reply:
Not all computers running at home have a GUI environment.  For years I didn't bother installing X on my Linux box as I only used it to run command line applications.  Some people run Raspberry Pis now in a similar configuration, to keep the install size down.  Also, I run Jamulus in Server mode with the GUI remotely, from my server, using remote X.

The phrase needs to be explicit in what it means: you need a graphical desktop environment installed on your OS - whether running locally on your desktop or remotely - to run Jamulus Server with a GUI.

Reply:
Microsoft also [suggests graphical environment](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/operating-environment-operating-system).

Reply:
I spent ten years using text-based computers exclusively. I like _home computer_. Sometimes technical writing can sacrifice precision to better reach a key audience. I'd like to be very clear here that running a server locally is something you can do "at home".

Reply:
> Jamulus can run in Server mode in the graphical environment of your home computer. This gives you a graphical user interface to control most of the settings

That sounds good to me.

Reply:
What motivates someone to run without the GUI on a graphical environment? Multiple parallel instances of the server?

Reply:
I'd generally run the server without the GUI if I were running it "permanently".  Spinning up a server for ad hoc use, probably I'd use the GUI.  I don't know about actual cases, though.